### PR TITLE
Added multi device support to U2F

### PR DESCRIPTION
**Pull request by mitsuhiko**

### CODE REVIEW

File: src/sentry/static/sentry/less/auth.less [hunk 1]
```diff
@@ -251,6 +252,22 @@ div.u2f-box {
...
+  div.device-failed {
+    display: none;
+    position: absolute;
+    top: 35%;
+    left: 50%;
+    width: 100px;
+    text-align: center;
+    margin-left: -50px;
+    &:after {
+      content: ":-(";
```
Comment (mattrobenolt) [inline src/sentry/static/sentry/less/auth.less]: 😦

**Comment by mattrobenolt** [comment]
I'm going to take this over and add a couple things:

* confirmation for removing key (will just use a `confirm()`)
* generate a better random name using our pet names library so they're more unique

File: src/sentry/static/sentry/app/components/u2finterface.jsx [hunk 2]
```diff
@@ -94,10 +120,37 @@ const U2fInterface = React.createClass({
...
 
+  renderFailure() {
+    let {deviceFailure} = this.state;
+    return (
+      <div className="failure-message">
+        <p><strong>{t('Error: ')}</strong> {{
+          'DEVICE_ERROR': t('Your U2F device reported an error.'),
+          'DUPLICATE_DEVICE': t('This device is already in use.'),
+          'UNKNOWN_DEVICE': t('The device you used for sign-in is unknown.'),
+          'BAD_APPID': t('The Sentry server administrator modified the ' +
```
Comment (mattrobenolt) [inline src/sentry/static/sentry/app/components/u2finterface.jsx]: This error message isn't very clear. Unless there are other ways of doing this, can we say something about the domain changing? "device registrations" isn't very clear to me what changed or how I'd fix the problem.

Comment (mitsuhiko) [inline src/sentry/static/sentry/app/components/u2finterface.jsx]: The message basically would need to indicate that you have to reach out to the Sentry server administrator / support.

**Pull request merged**